.vscode/tasks.json

```diff
@@ -34,7 +34,27 @@
 				"-f",
 				"net46",
 				"-o",
-				"${cwd}\\src\\Docker.PowerShell\\bin\\Module\\Docker",
+				"${cwd}\\src\\Docker.PowerShell\\bin\\Module\\Docker\\clr",
```

Comment:
This is getting quite complicated, to the point that you need vscode to build this thing. Is there no way to express this in project.json? If not, maybe we need a build.cmd script again... :(